acp/acp_local.go

```diff
@@ -106,13 +106,13 @@ func (l *ACPLocal) Close() error {
 
 func (l *ACPLocal) AddPolicy(
 	ctx context.Context,
-	creatorID string,
+	actorID string,
```

Comment:
todo: Don't forget to also update the creator in other places (like `acp/acp.go` and `source_hub_client.go` files)

Reply:
actually, I'm going to revert this change for now. It's not clear to me what "creator" even means in this context and what defra wants to do with it. ACP Core / SourceHub do not care about the actor creating the Policy. Maybe this could be removed entirely but that's not something I want to change in this PR. Once Andy resumes the integration with SourceHub we might have a better idea of what is desireable from the defra side. 
A thought regarding this is: if having the association between policy and the actor that created it is desireable from a Defra POV, maybe it could be added as part of the Policy attributes

Reply:
I am fine with that. We can change them once we have a more clearer picture.